users/assets/js/script.js

```diff
@@ -0,0 +1,303 @@
+document.addEventListener("DOMContentLoaded", function() {
+
+    var passwordInput = document.getElementById("password");
```

Comment:
Avoid multiple var variable declaration lines, declare multiple variable in one line separated by a comma. Please check all multiple variables declaration in this file.

Reply:
This is fixed